pages/en/lb4/Testing-your-application.md

```diff
@@ -65,18 +64,19 @@ Your `package.json` should then look something like this:
   // ...
   "devDependencies": {
     "@loopback/testlab": "^<current-version>",
+    "@types/mocha": "^<current-version>",
     "mocha": "^<current-version>"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha --recursive \"DIST/test\""
```

Comment:
Use of DIST requires the use of `lb-mocha` from `@loopback/build` which replaces `DIST` with the appropriate folder name based on version ... If you want to use mocha then it should be `dist` since we no longer support Node 6 (dist6).